server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -170,8 +169,10 @@ public void deleteGcCandidates(DataLevel level, Collection<String> paths) {
       }
       scanner.setRange(range);
 
-      return Iterators.transform(scanner.iterator(),
-          entry -> MetadataSchema.DeletesSection.decodeRow(entry.getKey().getRow().toString()));
+      return Iterators.transform(
```

Comment:
Instead of using more Guava code here, this could use Java streams. I wan't sure how to get the stream for a scanner so I experimented locally.   I ended up with the following that at least compiles.

```java
  return StreamSupport.stream(scanner.spliterator(), false)
    .filter(entry -> entry.getValue().equals(DeletesSection.SkewedKeyValue.NAME))
    .map(entry -> DeletesSection.decodeRow(entry.getKey().getRow().toString())).iterator();
```

Reply:
@keith-turner this also appears to work.  Why streams over iterators?  Curious.

Reply:
> Why streams over iterators?

I just think its nice to use functionality built into Java vs a third party library.  The Guava iterator utilities were created before Streams existed in Java.